docs/guidance/prometheus-grafana.md

```diff
@@ -66,6 +80,22 @@ kubectl get service
   * `# HELP`: Describe the meaning of this metric.
   * `# TYPE`: See [this document](https://prometheus.io/docs/concepts/metric_types/) for more details.
 
+* Three required environment variables are defined in [ray-cluster.embed-grafana.yaml](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-cluster.embed-grafana.yaml). See the Ray documentation ["Configuring and Managing Ray Dashboard"](https://docs.ray.io/en/latest/cluster/configure-manage-dashboard.html) for more details about these environment variables.
+  ```yaml
+  env:
+    - name: RAY_GRAFANA_IFRAME_HOST
+      value: http://127.0.0.1:3000
+    - name: RAY_GRAFANA_HOST
+      value: http://prometheus-grafana.prometheus-system.svc:80
+    - name: RAY_PROMETHEUS_HOST
+      value: http://prometheus-kube-prometheus-prometheus.prometheus-system.svc:9090
+  ```
+  * Note that we do not deploy Grafana in the head Pod, so we need to set both `RAY_GRAFANA_IFRAME_HOST` and `RAY_GRAFANA_HOST`. 
+    `RAY_GRAFANA_HOST` is used by the head Pod to send health-check requests to Grafana in the backend.
+    `RAY_GRAFANA_IFRAME_HOST` is used by your browser to fetch the Grafana panels from the Grafana server rather than from the head Pod.
+    Because we forward the port of Grafana to `127.0.0.1:3000` in this example, we set `RAY_GRAFANA_IFRAME_HOST` to `http://127.0.0.1:3000`.
```

Comment:
should we include instructions that if user does not want to port-forward, they should set up an ingress?

Reply:
We have instructions for setting up the ingress for the Ray dashboard. Setting up the ingress for Grafana seems to be beyond the scope of KubeRay. How about adding https://grafana.com/tutorials/run-grafana-behind-a-proxy/ to the doc?

Reply:
Add the link in https://github.com/ray-project/kuberay/pull/1278/commits/7362c589d64fd9a15ad3a135538c96ba60351c00. Does it make sense to you? Thx!